src/index.js

```diff
@@ -265,6 +265,28 @@ class HCaptcha extends React.Component {
       return hcaptcha.execute(captchaId, opts);
     }
 
+    setData (opts) {
+      const { captchaId } = this.state;
+
+      if (!this.isReady()) {
+        return;
+      }
+
+      if (opts && typeof opts !== "object") {
+        opts = null;
+      }
+
+      hcaptcha.setData(captchaId, opts);
+    }
+
+    getResponse() {
+      return hcaptcha.getResponse(this.state.captchaId);
+    }
+
+    getRespKey() {
```

Comment:
Make sure to update the [README.md section `Methods`](https://github.com/hCaptcha/react-hcaptcha#methods) with the newly exposed methods.

Reply:
Done, please confirm that my descriptions are accurate https://github.com/hCaptcha/react-hcaptcha/pull/122/commits/0107edbc01dc916d77583cbed7f133a44689cc56